notification/src/config/config.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+/* global process */
+require("dotenv").config();
+
+// to get the values from .env
+if(typeof process.env.NODE_ENV === "undefined" || process.env.NODE_ENV !== "production") {
```

Comment:
instead of this, can we change in the dotenv config file to read values from env variable, so that this check is not required